### PR TITLE
Implementing a base site map that includes all resources.

### PR DESCRIPTION
**Pull request by ironfroggy**

### CODE REVIEW

**Comment by ironfroggy** [comment]
Implements #980

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

**Comment by pkdash** [comment]
@hyi Can you review this?

File: hs_sitemap/templates/sitemap.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    h1,h2,h3,h4 {
+        margin: 20px;
+    }
+    h3 {
+        padding-left: 20px;
+    }
+    h4 {
+        padding-left: 40px;
+    }
+    </style>
```
Comment (hyi) [inline hs_sitemap/templates/sitemap.html]: @ironfroggy Is there a reason to put these style definitions in html template rather than a css file?

**Comment by hyi** [comment]
@ironfroggy It looks good to me. Just a simple comment for your consideration.

File: hs_sitemap/templates/sitemap.html (hunk 1 above)
Comment (ironfroggy) [inline hs_sitemap/templates/sitemap.html]: Simply wasn't a reason not to embed them for styles that are not applicable to any other pages, especially when this page serves a very specific, narrow case and is meant chiefly to be consumed by machines.

Comment (hyi) [inline hs_sitemap/templates/sitemap.html]: OK. Makes sense. +1 from me. You can merge it then.

**Pull request merged**